src/fsharp/fsi/fsi.fs

```diff
@@ -89,6 +93,7 @@ module internal Utilities =
 
     let ignoreAllErrors f = try f() with _ -> ()
 
+let referencedAssemblies = Dictionary<string, DateTime>()
```

Comment:
I'm not sure why Microsoft\visualfsharp has moved this out to be a global.  It looks wrong to do this for FCS.